[CI:DOCS] Update the Podman readme (#6120)
containers/libpod: README.md

* Support for pods to manage groups of containers together.
* Resource isolation of containers and pods.
* Support for a Docker-compatible CLI interface through Podman.
* Support for an HTTP API providing both a Docker-compatible interface and an improved interface exposing advanced Podman functionality.
* Integration with CRI-O to share containers and backend code.

Podman presently only supports running containers on Linux. However, we are building a remote client which can run on Windows and OS X and manage Podman containers on a Linux system via the HTTP API using SSH tunneling.

## Roadmap

> 1. Complete the Podman HTTP API and Podman v2, which will be able to connect to remote Podman instances via this API

Reviewer: I think I'd go with "REST API" over "HTTP API".

**[OCI Hooks Support](pkg/hooks/README.md)**
Information on how Podman configures [OCI Hooks][spec-hooks] to run when launching a container.

**[Podman API](http://docs.podman.io/en/latest/_static/api.html)**

> Documentation on the Podman HTTP API. Please note that the API is still in its early stages and not yet stable.

Reviewer: On the fence, but I'm leaning to keeping the varlink bit but marking it as to be deprecated with the V2 release.  It's still usable and supported atm...

Reviewer: oops, nvm, should have read further down

you to manage and maintain those images and containers in a production environment using
familiar container cli commands.  For more details, see the
[Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).

## Podman Legacy API (Varlink)

> Podman previously offered a Varlink-based API for remote management of containers.

Reviewer: ```suggestion
Podman offers a Varlink-based API for remote management of containers.
```

> This API is has been deprecated by the REST API.

Reviewer: ```suggestion
However, the Varlink API support is currently in maintenance mode and will be deprecated when Podman version 2.0 is released.  It will then be removed completely in a future release.
```

> Varlink support is in maintenance mode, and will be removed in a future release.

Reviewer: ```suggestion
```
